styles/variables-silicon.less

```diff
@@ -68,13 +68,15 @@
 
 // Item
 // ---------------------------------------
-@agate-item-font-size: 40px;
+@agate-item-font-size: 32px;
 @agate-item-font-weight: 500;
 @agate-item-label-font-size: 30px;
-@agate-item-side-outset: 32px;
-@agate-item-height: 90px;
+@agate-item-side-outset: 45px;
+@agate-item-top-bottom-outset: 26px;
```

Comment:
Please use multiples of 3 for all measurements. This avoids unpleasant and unpredictable positioning issues that arise when the browser tries to round values like `1/3` and `2/3`.